src/js/chartAuthoring.js

```diff
@@ -133,31 +153,77 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
             onChartAuthoringInterfaceReady: null,
             onPanelReady: null,
             onPieChartReady: null,
+            onSonifierReady: null,
             onToolReady: {
                 events: {
                     onPanelReady: "onPanelReady",
-                    onPieChartReady: "onPieChartReady"
+                    onPieChartReady: "onPieChartReady",
+                    onSonifierReady: "onSonifierReady"
                 },
                 args: ["{that}"]
-            }
+            },
+            onDataAppliedToDataEntryPanel: null
         },
         listeners: {
-            "onToolReady.addAriaConnections": "floe.chartAuthoring.addAriaConnections"
+            "onToolReady.addAriaConnections": "floe.chartAuthoring.addAriaConnections",
+            "onToolReady.bindPlayClick": {
+                "this": "{that}.dom.sonifierPlay",
+                "method": "click",
+                "args": ["{chartAuthoringInterface}.sonifier.events.onSonificationRequested.fire"]
+            },
+            "onToolReady.bindStopClick": {
+                "this": "{that}.dom.sonifierStop",
+                "method": "click",
+                "args": ["{chartAuthoringInterface}.sonifier.stopSonification"]
+            }
         },
-        invokers:{
-            "updateDataEntryPanel": {
-                funcName: "floe.chartAuthoring.updateDataEntryPanel",
+        invokers: {
+            "updateDataEntryPanelFromDataSet": {
+                funcName: "floe.chartAuthoring.updateDataEntryPanelFromDataSet",
                 args: ["{that}", "{arguments}.0"]
             },
             "resetDataEntryPanel": {
-                funcName: "floe.chartAuthoring.updateDataEntryPanel",
+                funcName: "floe.chartAuthoring.updateDataEntryPanelFromDataSet",
                 args: ["{that}", []]
+            },
+            "updateActiveElements": {
+                funcName: "floe.chartAuthoring.updateActiveElements",
+                args: ["{that}"]
+            },
+            "highlightActiveLegendRow": {
+                funcName: "floe.chartAuthoring.highlightChange",
+                args: ["{that}", "{that}.chartAuthoringInterface.pieChart.legend.rows"]
+            },
+            "highlightActivePieSlice": {
+                funcName: "floe.chartAuthoring.highlightChange",
+                args: ["{that}", "{that}.chartAuthoringInterface.pieChart.pie.paths"]
+            }
+        },
+        model: {
```

Comment:
Hi @cindyli - I think this makes sense as a delegation of responsibility and will better decouple the components involved. Given @amb26's feedback on further modelization of the play highlighting behaviour, I may roll this into whatever refactoring I do to support that. Make sense?